spec/observables/defer-spec.ts

```diff
@@ -132,7 +132,7 @@ describe('defer', () => {
       const source = hot('--a--b--c--|');
       const sourceSubs = '^-----!     ';
       const expected = '  --a--b-     ';
-      const unsub =      '------!     ';
+      const unsub = '------!     ';
```

Comment:
This marble diagram wasn't aligned with spaces within the literal, so Prettier messed up its alignment:
```suggestion
      const unsub = '     ------!     ';
```